### PR TITLE
Adds the r-rcpp recipe.

### PR DESCRIPTION
**Pull request by jrderuiter**
Adds the r-rcpp recipe (adhering to the guidelines in https://github.com/conda-forge/staged-recipes/issues/2009).

### CODE REVIEW

**Comment by johanneskoester** [comment]
Great! Do you mind removing the skeleton comments?

**Comment by jrderuiter** [comment]
@johanneskoester No problem. Should we support windows or is it ok to skip windows builds? Appveyor is currently erroring on: 

```
Package missing in current win-32 channels: 
  - r-base 3.3.2*
```

**Comment by jrderuiter** [comment]
Ready to merge I guess!

File: recipes/r-rcpp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('Rcpp')"  # [not win]
+    - "\"%R%\" -e \"library('Rcpp')\""  # [win]
+
+about:
+  home: http://www.rcpp.org, http://dirk.eddelbuettel.com/code/rcpp.html, https://github.com/RcppCore/Rcpp
+  license: GPL (>= 2)
```
Comment (jakirkham) [inline recipes/r-rcpp/meta.yaml]: If this is GPL2, just say `GPL 2.0` or similar.

File: recipes/r-rcpp/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  summary: The 'Rcpp' package provides R functions as well as C++ classes which offer a seamless
+    integration of R and C++. Many R data types and objects can be mapped back and forth
+    to C++ equivalents which facilitates both writing of new code as well as easier
+    integration of third-party libraries. Documentation  about 'Rcpp' is provided by
+    several vignettes included in this package, via the  'Rcpp Gallery' site at <http://gallery.rcpp.org>,
+    the paper by Eddelbuettel and  Francois (2011, JSS), and the book by Eddelbuettel
+    (2013, Springer); see  'citation("Rcpp")' for details on these last two.
+  license_family: GPL3
```
Comment (jakirkham) [inline recipes/r-rcpp/meta.yaml]: This clashes with GPL2 above. Also can just say `GPL`.

File: recipes/r-rcpp/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/r-rcpp/meta.yaml]: Is there a license file we can include?

ref: https://conda.io/docs/building/meta-yaml.html#license-file

File: recipes/r-rcpp/build.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+mv DESCRIPTION DESCRIPTION.old
+grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
+
+$R CMD INSTALL --build .
+
+# Add more build steps here, if they are necessary.
+
+# See
+# http://docs.continuum.io/conda/build.html
+# for a list of environment variables that are set during the build process.
```
Comment (jakirkham) [inline recipes/r-rcpp/build.sh]: Please remove the skeleton comments from here.

File: recipes/r-rcpp/meta.yaml (hunk 1 above)
Comment (jrderuiter) [inline recipes/r-rcpp/meta.yaml]: No license file as far as I can see. I'll change the version.

**Pull request merged**